docs/releases/1.10.2.txt

```diff
@@ -25,3 +25,7 @@ Bugfixes
 
 * Fixed a crash in ``runserver`` logging during a "Broken pipe" error
   (:ticket:`27271`).
+
+* Fixed a regression in ``ModelAdmin.construct_change_message()`` when
```

Comment:
I would say, "Fixed a regression where unchanged localized date/time fields were listed as changed in the admin's model history messages", otherwise LGTM.